tools/cabana/chart/chart.cc

```diff
@@ -215,7 +215,13 @@ void ChartView::updateTitle() {
   }
   for (auto &s : sigs) {
     auto decoration = s.series->isVisible() ? "none" : "line-through";
-    s.series->setName(QString("<span style=\"text-decoration:%1\"><b>%2</b> <font color=\"gray\">%3 %4</font></span>").arg(decoration, s.sig->name, msgName(s.msg_id), s.msg_id.toString()));
+    QColor windowText = palette().color(QPalette::WindowText);
+    auto titleColor = windowText.name(QColor::HexArgb);
+    windowText.setAlpha(180);
+    auto msgColor = windowText.name(QColor::HexArgb);
```

Comment:
does it make sense to move these declarations out of the loop?

Reply:
@incognitojam Good idea, have done!